WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -2,7 +2,11 @@
 
 <% if @competition.championships.present? %>
   <h2 class="alert">This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %></h2>
-  <% end %>
+<% end %>
+
+<% @competition.championship_warnings.each do |key, warning| %>
+  <h2 class="alert"><%= warning %></h2>
```

Comment:
If in the end this is a sentence only (as suggested above,) maybe it would look better if it went right after the sentence above instead of on a new line?

Reply:
Having it on a new line would give more pop that there is an additional warning about this, instead of seeming like it is part of the championship text.